ebuild-writing/variables/text.xml

```diff
@@ -600,55 +718,37 @@ table.
 </p>
 
 </body>
+</subsection>
 </section>
 
 <section>
-<title>ROOT</title>
+<title>Variables reserved for the package manager</title>
```

Comment:
Small nit on the title: it doesn't seem to be reserved variable names, more like restrictions on variable and function names.

Reply:
It's "Reserved commands and variables" in PMS. We could simply copy this (then again, it is the variables section).